### ISSUE TITLE
DS - /feature/ - Visuals for Community Sentiment Data

### ISSUE THREAD

**Issue by sayerjk** [BloomTech-Labs/underdog-devs-ds-a#142]
Picking up where Roger and Bradley left off with a current [issue](https://bloomtechlabs.atlassian.net/browse/BL-134?atlOrigin=eyJpIjoiNWRlZWIwOWQ3NzZkNGQ5MTg0N2I0NjY4ZjgwNTZmOWMiLCJwIjoiaiJ9) on Jira. We would like to explore this new data, create visualizations, and ensure we can receive new user input (through collaboration with Web students).

**Comment by jinjahninjah** [comment]
We might be able to add this into the current [issue](https://bloomtechlabs.atlassian.net/browse/BL-134?atlOrigin=eyJpIjoiNWRlZWIwOWQ3NzZkNGQ5MTg0N2I0NjY4ZjgwNTZmOWMiLCJwIjoiaiJ9) in the subtasks? Thoughts?

**Comment by sayerjk** [comment]
I wasn't sure how we should split it, or if Roger and Bradley needed the Issue to go through in order to graduate. 

I saw the distinction as our ticket dealing with sending processed data to FE/BE and theirs dealing with receiving new data from FE/BE. I also saw the potential for various pull requests, if we are dealing with exploratory data analysis (notebook), adding the visualizations into the app, and creating an endpoint so Web can pull visuals from DS/Mongo. All the while communicating and documenting changes for clarity across DS, FE, and BE.
